products/compute/api.yaml

```diff
@@ -12966,14 +12966,24 @@ objects:
             values:
               - :EXCLUDE_ALL_METADATA
               - :INCLUDE_ALL_METADATA
+              - :CUSTOM_METADATA
             default_value: :INCLUDE_ALL_METADATA
+          - !ruby/object:Api::Type::Array
```

Comment:
Does order matter here? If not, I'd recommend marking this as `is_set: true` in terraform.yaml

Reply:
Set this value to make it treated as a set